### PR TITLE
Change org/module name

### PR DESCRIPTION
**Pull request by kalaiyarasiganeshalingam**
## Purpose
$Subject

## Automation tests
 - Unit tests 
   > Code coverage information
 - Integration tests
   > Details about the test cases and coverage

## Security checks
 - Followed secure coding standards in http://wso2.com/technical-reports/wso2-secure-engineering-guidelines? yes/no
 - Ran FindSecurityBugs plugin and verified report? yes/no
 - Confirmed that this PR doesn't commit any keys, passwords, tokens, usernames, or other secrets? yes/no

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -14,7 +14,7 @@ The following sections provide you with information on how to use the Ballerina
 
 | Ballerina Language Versions  | Google Spreadsheet API Version |
 |:---------------------------:|:------------------------------:|
-|  1.0.x, 1.1.x                     |   V4                           |
+|  1.0.x, 1.1.x, 1.2.x                    |   V4                           |
```
Comment (chamil321) [inline README.md]:
```suggestion
|  1.0.x, 1.1.x, 1.2.x             |   V4                           |
```

File: src/googleapis.sheets4/Module.md [hunk 2]
```diff
@@ -23,7 +23,7 @@ The `wso2/gsheets4` module allows you to perform following operations.
 
 |                             |       Versions               |
 |:---------------------------:|:---------------------------:|
-| Ballerina Language          | 1.0.x, 1.1.x                     |
+| Ballerina Language          | 1.0.x, 1.1.x, 1.2.0                     |
```
Comment (chamil321) [inline src/googleapis.sheets4/Module.md]:
```suggestion
| Ballerina Language          | 1.0.x, 1.1.x, 1.2.0              |
```

File: src/googleapis.sheets4/Module.md [hunk 3]
```diff
@@ -23,7 +23,7 @@ The `wso2/gsheets4` module allows you to perform following operations.
 
 |                             |       Versions               |
 |:---------------------------:|:---------------------------:|
-| Ballerina Language          | 1.0.x, 1.1.x                     |
+| Ballerina Language          | 1.0.x, 1.1.x, 1.2.0         |
```
Comment (daneshk) [inline src/googleapis.sheets4/Module.md]: This is only compatible with 1.2.0 right?

Comment (kalaiyarasiganeshalingam) [reply src/googleapis.sheets4/Module.md]: I haven't change anything in the implementation level therefore we can keep other version as well.

**Pull request merged**